.quickstart/quickstart.yml

```diff
@@ -0,0 +1,16 @@
+database_key: iterable_database
+schema_key: iterable_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  iterable__using_campaign_label_history:
+    - campaign_label_history
+  iterable__using_campaign_suppression_list_history:
+    - campaign_suppression_list_history
+    
```

Comment:
Seems like this was left out--

    iterable__using_user_unsubscribed_message_type_history:
        - user_unsubscribed_message_type

Reply:
Good catch, just updated

Reply:
Oh actually I just re checked and it looks like we have logic that checks for both `user_unsubscribed_message_type_history` and `user_unsubscribed_message_type`, so we should include both

Reply:
@fivetran-reneeli `user_unsubscribed_message_type` is actually deprecated so I intentionally removed it. This is something we can remove down the line, but for quickstart we should exclude it.